src/App.vue

```diff
@@ -11,6 +15,20 @@ export default {
   name: 'App',
   components: {
     ProblemPractice
+  },
+  data() {
+    return {
+      number: 0,
+    }
+  },
+  methods: {
+    updateNumber(value) {
+      if (value)
```

Comment:
It doesn't work. Value was coming in as NaN because it was an empty string in the input, but I was trying to parse it as a number before emitting it to App.vue. I moved the parsing to App.vue's updateNumber to avoid this. Does that work?

Reply:
Yeah I like that better :)